### PR TITLE
added Java Calendar isLenient() method

### PR DESCRIPTION
**Pull request by ATREAY**
### Description

added Java Calendar isLenient() method(This pull request pertains to issue #3103)

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x]  I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/isLenient/isLenient.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+> **Note:** When a `Calendar` is lenient, it allows date and time values that may not make sense in a real-world context. For example, `February 30th` or a time value of `25:70:90` would be accepted. When a `Calendar` is non-lenient, it enforces stricter rules, and attempting to set invalid date or time values will throw exceptions, such as `IllegalArgumentException`.
+
+
+## Example 1
+
+The example below demonstrates the use of the `.isLenient()` method.
+
+```java
+
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isLenient/isLenient.md:30]: remove empty line 30

File: content/java/concepts/calendar/terms/isLenient/isLenient.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import java.util.Calendar;
+
+public class CalendarIsLenientExample {
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isLenient/isLenient.md:33]: the code should use 2 spaces for indentations not tabs

File: content/java/concepts/calendar/terms/isLenient/isLenient.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+	}
+}
+```
+
+This code will return an output similar to the following:
+
+```shell
+Current Calendar: Wed Sep 06 17:51:57 GMT 2023
+Calendar is lenient: true
+```
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isLenient/isLenient.md:57]: add and empty line between the shell block and the example 2 header

File: content/java/concepts/calendar/terms/isLenient/isLenient.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+## Example 2
+
+The example below demonstrates the use of the `.isLenient()` method.
+
+```java
+import java.util.Calendar;
+
+public class CalendarIsLenientExample{
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/isLenient/isLenient.md:65]: update the indentations

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@ATREAY I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by ATREAY** [comment]
Hey @caupolicandiaz,
I have implemented the necessary modifications. Please go through it.

**Review by caupolicandiaz — APPROVED** [review]
@ATREAY Looks good. This is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@ATREAY I made a few edits, fixing the modification of the `yarn.lock` and `package.json` files. Now it looks good to merge.

**Comment by SSwiniarski** [comment]
@ATREAY also, for future reference, it is not recommended that you submit PRs from your main branch. It is better to create a separate branch off of main for each PR.

**Comment by ATREAY** [comment]
I appreciate your guidance, @SSwiniarski. 
I will diligently incorporate all the recommendations you have provided into my future Pull Requests (PRs).

**Pull request merged**